src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/gui/MixinChatScreen.java

```diff
@@ -56,6 +55,7 @@ private void handleChatMessage(String chatText, boolean addToHistory, CallbackIn
         ChatSendEvent chatSendEvent = new ChatSendEvent(chatText);
 
         EventManager.INSTANCE.callEvent(chatSendEvent);
+        minecraft.gui.getChat().discardDraft();
```

Comment:
[discardCraft] is usually called automatically when [shouldDiscardDraft] is true.

```java
protected boolean shouldDiscardDraft() {
		return this.exitReason != ChatScreen.ExitReason.INTERRUPTED
			&& (this.exitReason != ChatScreen.ExitReason.INTENTIONAL || !this.minecraft.options.saveChatDrafts().get());
	}
```

```java
if (keyEvent.isConfirmation()) {
			this.handleChatInput(this.input.getValue(), true);
			this.exitReason = ChatScreen.ExitReason.DONE;
			this.minecraft.setScreen(null);
			return true;
		}

```

Might need to compare with Vanilla behavior first to verify.